SqlScriptDom/Parser/TSql/TSql130.g

```diff
@@ -25173,16 +25173,24 @@ tableHashDistributionPolicy returns [TableHashDistributionPolicy vResult = Fragm
 	Identifier vIdentifier;
 }
 	:
-		tHash:Identifier LeftParenthesis vIdentifier = identifier
-		{
-			Match(tHash, CodeGenerationSupporter.Hash);
-			vResult.DistributionColumn = vIdentifier;
-			UpdateTokenInfo(vResult, tHash);
-		}
-		tRParen:RightParenthesis
-		{
-			UpdateTokenInfo(vResult, tRParen);
-		}
+		tHash:Identifier
+        {
```

Comment:
Think the indentation here got messed up

Reply:
I think it's how Github displays the code in the pr, because when I view it locally or in codespaces the indentation looks ok.